write_to_score.py

```diff
@@ -31,29 +33,76 @@ def write_tsvs_to_scores(
+        staff.
+        By default, the staves indicated in the 'staff' column of the labels are used, or, if
+        such a column is not present, labels will be inserted under the lowest staff -1.
+
+    voice : Optional[Literal[1, 2, 3, 4]]
+        If you pass the ID of a notational layer (where 1 is the upper voice, blue in MuseScore),
+        the labels will be attached to that one.
+        By default, the notational layers indicated in the 'voice' column of the labels are used,
+        or, if such a column is not present, labels will be inserted for voice 1.
+
+    harmony_layer : Optional[Literal[0, 1, 2]]
+        | By default, the labels are written to the layer specified as an integer in the
+          'harmony_layer' of the label. Pass an integer to select a particular layer:
+        | * 0 to attach them as absolute ('guitar') chords, meaning that when opened next time,
+        |   MuseScore will split and encode those beginning with a note name
+        |   (resulting in ms3-internal harmony_layer 3).
+        | * 1 the labels are written into the staff's layer for Roman Numeral Analysis.
+        | * 2 to have MuseScore interpret them as Nashville Numbers
+
+    check_for_clashes : bool
+        By default, warnings are thrown when there already exists a label at a position
+        (and in a notational layer) where a new one is attached. Pass False to deactivate these
+        warnings.
+
+
     """
     output_tsv_dir = Path(output_tsv_dir)
     annotations_base_dir = Path(annotations_base_dir)
-
-    output_paths = sorted(glob(str(output_tsv_dir / "**" / "*.tsv"), recursive=True))
-    output_paths = [
-        path
-        for path in output_paths
-        if not path.endswith("_results.tsv") and not path.endswith("results_midi.tsv")
-    ]
-    for piece_name in tqdm(output_paths, desc="Writing labels to scores"):
-        piece_name = Path(piece_name).relative_to(output_tsv_dir)
-        try:
-            eu.write_labels_to_score(
-                output_tsv_dir / piece_name.parent,
-                annotations_base_dir if fuzzy else annotations_base_dir / piece_name.parent,
-                piece_name.stem,
-            )
-        except Exception:
-            logging.exception("Error writing score out to %s", output_tsv_dir / piece_name.parent)
+    outputs_are_under_corpus_path = (output_tsv_dir == annotations_base_dir) or (
+        annotations_base_dir in annotations_base_dir.parents
+    )
+    corpus = ms3.Corpus(annotations_base_dir, only_metadata_fnames=False)
+    if not outputs_are_under_corpus_path:
+        # if the Corpus directory itself includes any labels we exclude them to avoid ambiguity
+        if any(x.is_dir() and x.name == "labels" for x in annotations_base_dir.iterdir()):
+            excluded_labels_path = annotations_base_dir / "labels"
+        corpus.view.exclude("path", str(excluded_labels_path))
```

Comment:
Error here if the above if is False. Probably this just needs to be indented?

Reply:
Indeed, sorry